Meta/Azure/Lagom.yml

```diff
@@ -213,5 +212,6 @@ jobs:
           UBSAN_OPTIONS: 'print_stacktrace=1:print_summary=1:halt_on_error=1'
 
     - script: |
+        CCACHE_DIR='$(SERENITY_CCACHE_DIR)' ccache --evict-older-than 1d
         CCACHE_DIR='$(SERENITY_CCACHE_DIR)' ccache -s
-      displayName: 'Cache Stats'
+      displayName: 'Cache Prune and Stats'
```

Comment:
nit: "Cache Prune and Stats" sounds quite awkward (here and below). Maybe: "Prune and Show Cache Stats"

Reply:
That sounds like we're pruning the cache stats. "Prune Cache, and Show Stats"? (... could this be two steps, one for pruning, one for showing stats?)

Reply:
I split out all new commands into a new (appropriately named) step.